glossary.yml

```diff
@@ -734,7 +734,12 @@
       Mbinu ya ukuzaji programu ambayo inasisitiza hatua nyingi ndogo na maoni 
       endelevu badala ya mipango ya mbeleni na ratiba ya muda mrefu. 
       [Exploratory programming](#exploratory_programming) mara nyingi huwa chapa.
-
+  uk:
+    term: "гнучка розробка"
+    def: >
+      Методологія розробки програмного забезпечення, яка робить акцент на малих 
```

Comment:
@Herobread А як щодо варіанту "акцент на поступових змінах" ?

Reply:
@olexandr-konovalov agreed, updated, please review

Reply:
Мабуть всеж ітераціях -> змінах ?

порівняйте з https://sum20ua.com/?page=1253&searchWord=%D1%96%D1%82%D0%B5%D1%80%D0%B0%D1%86%D1%96%D1%8F&wordid=181736 

Я б спочатку зрозумів це як ітерації у циклі, наприклад

Reply:
так, `ітерація` - краще в математичному контексті

змінив на `зміна`

Reply:
Thanks @Herobread ! How does it look to you @YehorBoiar ?

Reply:
When I imagine an agile methodology, I always think of this picture. 

![image](https://github.com/carpentries/glosario/assets/94189697/88271558-faee-4917-8961-2ad60d00f02a)

Also, I did some research on agile methodology, and all papers that I have found use the word "iteration" rather than "change" or similar. Here is a paper that I found the most topical:

- [Adoption_of_agile_methodology_in_software_development.pdf](https://github.com/user-attachments/files/15951635/Adoption_of_agile_methodology_in_software_development.pdf) [2]

Paper has a clear description of what agile methodology is on page 2, and it uses the word iteration. 

Therefore, I think that the word "ітерація" would be better.

Reply:
interesting, I agree with `ітерація` as it is quite popular in math and CS, but at the same time, if you choose by meaning of it in vocabulary, then `зміна` is better

Reply:
Well, they have taken this term from the "Гірничий енциклопедичний словник" that had its last update in 2004, so I don't think that the term "ітерація" is up to date. In the [cambrige dictionary](https://dictionary.cambridge.org/us/dictionary/english/iteration)  it has 2 definitions:
- "the [process](https://dictionary.cambridge.org/us/dictionary/english/process) of doing something again and again, usually to [improve](https://dictionary.cambridge.org/us/dictionary/english/improve) it, or one of the [times](https://dictionary.cambridge.org/us/dictionary/english/times) you do it"
- "an [amount](https://dictionary.cambridge.org/us/dictionary/english/amount) that you get when you use a [mathematical](https://dictionary.cambridge.org/us/dictionary/english/mathematical) [rule](https://dictionary.cambridge.org/us/dictionary/english/rule) several [times](https://dictionary.cambridge.org/us/dictionary/english/times)"

And I think that we could use the word iteration in the first meaning of its word according to cambrige dictionary, and here are some papers that actually do that:
- https://nzlubp.org.ua/index.php/journal/article/view/400/377 [page 2]
- http://dspace.wunu.edu.ua/bitstream/316497/42578/1/%d0%91%d0%b0%d0%bb%d0%b0%d0%b7%d1%8e%d0%ba.pdf [page 3]
- https://archive.mcnd.org.ua/index.php/conference-proceeding/article/view/980/1000 [page 2]

Reply:
@YehorBoiar you almost swayed me into "iteration",  but that did not last long (especially since we clarify about the feedback in the following text). Labeled this as "question" for the next call.

Reply:
One more though for "iterations" vs "changes": "changes" looks like changing what's already there. "iterations" also covers adding new stuff. Let's wait till the group call to discuss.